diagnostic.md

```diff
@@ -40,7 +81,7 @@ like to show all movies favorited by a profile on
 `http://localhost:3000/profiles/1`
 
   ```md
-    # < Your Response Here >
+    It helps ensure that the object is correctly created
```

Comment:
A better answer would be: "A serializer controls what data is returned from a request."